### ISSUE TITLE
Eclipse run configurations do not appear in GUI when imported

### ISSUE THREAD

**Issue by TwilightFlower** [FabricMC/fabric-loom#111]
the eclipse run configurations don't show up in the GUI when imported, making them unusable. (ironically, the one place they DO show up is in the export menu). this can be fixed by deleting

    <listAttribute key="org.eclipse.debug.core.MAPPED_RESOURCE_PATHS">
        <listEntry value="/Minecraft Server"/>
    </listAttribute>
    <listAttribute key="org.eclipse.debug.core.MAPPED_RESOURCE_TYPES">
        <listEntry value="4"/>
    </listAttribute>

from the start of the .launch file.

**Comment by Sollace** [comment]
Nice. I'll have to try this.

The fix I had before was to disable the feature in eclipse that filters out run configs for damaged/unloaded projects.

**Comment by modmuss50** [comment]
Any update on this? It seems like its been an issue since I blindly added eclipse support but still no one has given me a solution.

**Comment by Sollace** [comment]
@modmuss50 I can confirm, the fix is to change these lines:

//projectname_server.launch
```
    <listAttribute key="org.eclipse.debug.core.MAPPED_RESOURCE_PATHS">
        <listEntry value="/Minecraft Server"/>
    </listAttribute>
```
//projectname_client.launch
```
    <listAttribute key="org.eclipse.debug.core.MAPPED_RESOURCE_PATHS">
        <listEntry value="/Minecraft Client"/>
    </listAttribute>
```

Both to:
```
    <listAttribute key="org.eclipse.debug.core.MAPPED_RESOURCE_PATHS">
        <listEntry value="/projectname"/>
    </listAttribute>
```

Where `projectname` is the name of the eclipse project and root directory/project folder. HDSkins in this example:
![image](https://user-images.githubusercontent.com/6429283/64554607-0cef0400-d33c-11e9-9cbe-99c272823228.png)

**Comment by modmuss50** [comment]
Should be finally fix in 0.2.7 loom, please test. If it works ill backport the fixes.
